app/utils/UserAnswersValidation.scala

```diff
@@ -0,0 +1,710 @@
+/*
```

Comment:
I think this file is too large. We could split this out into more modular code per event, it would probably make maintaining the code easier down the line.